src/engraving/dom/hairpin.cpp

```diff
@@ -544,4 +534,30 @@ String Hairpin::accessibleInfo() const
     }
     return rez;
 }
+
+//---------------------------------------------------------
+//   translatedSubtypeUserName
+//---------------------------------------------------------
+
+String Hairpin::translatedSubtypeUserName() const
```

Comment:
Here (and for HairpinSegment) too, isn't it better to implement `subtypeUserName` rather than only `translatedSubtypeUserName`?